mirrord/layer/src/lib.rs

```diff
@@ -626,7 +630,10 @@ pub(crate) unsafe extern "C" fn fork_detour() -> pid_t {
             let new_connection = ProxyConnection::new(
                 parent_connection.proxy_addr(),
                 NewSessionRequest::Forked(parent_connection.layer_id()),
-                PROXY_CONNECTION_TIMEOUT,
+                PROXY_CONNECTION_TIMEOUT
+                    .get()
+                    .copied()
+                    .unwrap_or_else(|| Duration::from_secs(31536000)),
```

Comment:
Should be initialized, I'd just do `.get().expect(...)`, so we don't need this value hardcoded here